content/sources/workflow.adoc

```diff
@@ -2,7 +2,7 @@
 
 eXo Platform product relies on a set of Git repositories :
 
-* https://github.com/Meeds-io[*meeds-io*]: Meeds-io association product Git organization. eXo Platform still the main contributor and maintainer of the source code provided by the association. Thus, we will use the same process for this repository as any other eXo Platform repository.
+* https://github.com/Meeds-io[*meeds-io*]: Meeds-io association product Git organization. Meeds Lab is the main contributor of the source code provided by the association. eXo Platform is the main maintener of this source code (the eXo Maintenance Team is in charge to fix bug on maintenance versions)
```

Comment:
IMO, here just need to describe the repository as the property of https://meed.io association with License LGPL v3 (Open Source and free to use). In addition we will need to remind that all members of the association benefits from this product and thus any feature/fix would affect the others. That's why we will need to communicate feature proposals and fixes through [MIPs](https://github.com/Meeds-io/MIPs) and [issues](https://github.com/Meeds-io/meeds/issues)

Reply:
+1

Reply:
done